### PR TITLE
Update from "Feedback" to "Jetpack Feedback"

### LINKED ISSUE #9637: Privacy: Change "Feedback" to "Jetpack Feedback" or something
#### Steps to reproduce the issue

We will soon be displaying the privacy exporter and eraser friendly names to the admin. The current name used for grunion feedback is simply "Feedback" - which might not be informative enough to the administrator. Recommend "Jetpack Feedback" or something.

#### What happened instead

export

### PR DESCRIPTION
**Pull request by Rahmon**
Fixes #9637

#### Changes proposed in this Pull Request:

*  Update from "Feedback" to "Jetpack Feedback"

image

### CODE REVIEW

File: modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -151,11 +151,12 @@ function __construct() {
 		// custom post type we'll use to keep copies of the feedback items
 		register_post_type( 'feedback', array(
 			'labels'            => array(
-				'name'               => __( 'Feedback', 'jetpack' ),
+				'name'               => __( 'Jetpack Feedback', 'jetpack' ),
 				'singular_name'      => __( 'Feedback', 'jetpack' ),
 				'search_items'       => __( 'Search Feedback', 'jetpack' ),
 				'not_found'          => __( 'No feedback found', 'jetpack' ),
 				'not_found_in_trash' => __( 'No feedback found', 'jetpack' ),
+				'menu_name'          => __( 'Feedback' )
```
Comment (jeherve) [inline modules/contact-form/grunion-contact-form.php]: Could you add the `jetpack` textdomain here, so that string can be translated? 

Thank you!

**Comment by jeherve** [comment]
While this works, I fear it may introduce more problems down the line. For example, since that code will be shared and used on WordPress.com sites as well, you will see "Jetpack Feedback" as the page title when viewing your Feedback in wp-admin and in Calypso:

69b047305c3bafbf195f78a74e6a2dc2 _screenshot 2018-09-07 at 16 02 51

@dereksmart What's your take on this?

**Comment by brbrr** [comment]
@Automattic/jetpack-design any suggestions regarding Jeremy's comment above?

**Comment by rickybanister** [comment]
I'm not too well versed in this issue, but if we want the title to be more descriptive in the export area, something like 'Customer Feedback' would be more helpful. If it needs to tell you that it's via jetpack, then 'Customer Feedback (Jetpack)' would be consistent with our widgets and things. 

I do not think we should change the menu title or page title though.

**Comment by MichaelArestad** [comment]
I wonder if we should start tucking Feedback into the Jetpack menu in the main nav.

**Comment by rickybanister** [comment]
Does that become a larger existential question about Jetpack's information architecture then? It also provides other custom post types and hooks into almost every part of WordPress. Will we end up putting an entire copy of WordPress inside of the Jetpack menu (to go to the logical extreme)?

**Comment by jeherve** [comment]
> I wonder if we should start tucking Feedback into the Jetpack menu in the main nav.

If we choose to go in that direction, we must consider the other plugin that currently hooks into that existing Feedback menu: the Polldaddy plugin that's installed on all WordPress.com sites and can be installed on self-hosted sites adds itself to the Feedback menu item, so if we were to remove that menu item we would have to make changes to the Polldaddy plugin first.

screenshot 2018-10-16 at 21 54 38

**Comment by MichaelArestad** [comment]
> Does that become a larger existential question about Jetpack's information architecture then? It also provides other custom post types and hooks into almost every part of WordPress. Will we end up putting an entire copy of WordPress inside of the Jetpack menu (to go to the logical extreme)?

It does. I don't know the answer here. Perhaps Woo is heading in the right direction with their full screen app?

**Comment by jeherve** [comment]
I'll close this PR for now because of the lack of activity on this. We can always reopen in the future if needed, but it will need a rebase, so it may be easier to start a new PR at this point.

**Pull request closed without merging**